src/components/TimeseriesChart/TimeseriesChart.tsx

```diff
@@ -107,6 +127,19 @@ export class TimeseriesChart extends React.Component<
     }
   };
 
+  onMouseMove = (data: any) => {
```

Comment:
Can we define type here?

Reply:
Currently the `onMouseMove` prop of `TimeseriesChart` has the type `(e: any) => void` - so used the same here for backward compatibility.